.nox/design/allnoxtypes.entity.nox.yaml

```diff
@@ -19,17 +19,18 @@ keys:
 attributes:
```

Comment:
Let's rearrange the types a bit, because there are SimpleTypes after line 83 (which is marked as CompoundType section). Ideally, it should wollow the same order as in NoxTypes.cs (which should also be rearranged).

Reply:
Should we also add the currencyCode3 field?

Reply:
done